app/views/greetings_api/random_greeting.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>GreetingsApi#random_greeting</h1>
+<p>Find me in app/views/greetings_api/random_greeting.html.erb</p>
```

Comment:
- [ ] according to the requirements you should Create a static view that will be the root of your app, it should only contain `<div id="root"> `